aiken-contracts/fractional/README.md

```diff
@@ -55,21 +55,43 @@ pub type Datum {
 }
 ```
 
-The bundle token information needs to match what is on the UTxO and the cost token information can be whatever token the seller wishes to be paid in. The maximum bundle size is a hardcoded maximum of 10 bundles. This is an arbitrary pick and can be changed to whatever is needed in production.
+The bundle token information needs to match what is on the UTxO and the cost token information can be whatever token the seller wishes to be paid in for a completed sale.
 
-A bundle sale with a bundle size of 100 and a cost of 10 ADA.
+### Example Sale
+A bundle sale with a bundle size of 100 tokens and a cost of 10 ADA. On the UTxO for the sale there will be
 
 ```bash
 1000 policy.token_name
 ```
 
-A buyer wants 6 bundles. They will receive 600 tokens at a cost of 60 ADA.
+A buyer wants 6 bundles. They will receive 600 tokens at a cost of 60 ADA. The remaining 400 tokens will be sent back to the contract for another sale and the seller will receive 60 ADA.
 
 ```bash
 # script return
 400 policy.token_name
-# buyer pay
+# buyer payout
 600 policy.token_name
-# seller pay
+# seller payout
 60 ADA
-```
\ No newline at end of file
+```
+
+## Staking
+
+The staking contract can only be registered and redelegated. It can not be deregistered. The only pool the stake script is allowed to redelegate to is the pool defined in the data contract. The stake credential will accumulate rewards over time and at any point may be rewarded the ADA to the reward address defined in the data contract. Because of the min utxo required, rewards below the minimum can not be taken out as the contract expects an exact payout of the reward.
```

Comment:
What is the reason the pool delegated to cannot be changed?

Reply:
nevermind. It looks like you just have to first update the reference data contract and then re-delegate.